std_msgs/Header header
bool foot_r
bool foot_l
bool heel_r
bool heel_l
